minute-to-sec-convertor.js

```diff
@@ -1,12 +1,8 @@
-/**
- * @param {number} minutes to be raise
- * @param {number} minutes must be an natural number
- * @return {number} returns seconds of a given minutes
-*/
+

```

Comment:
/**
 * convert minutes to seconds
 * @param {number} minutes - minutes must be an natural number
 * @returns {number} - returns seconds of a given minutes
 */